src/TestHelper/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I want to focus on the remote execution so I would to keep the changes in this PR to the remote execution only and we can later move more test code to PlatformDetection there.

I kept the test library folder with other libraries per @ericstj recommendation. so I am not sure if we have to move it to the common folder?  please advise if I should do that. 

for naming I'll go with CoreFX.Private.TestUtlities. let me know if you disagree with this name.